xep-0234.xml

```diff
@@ -218,25 +381,33 @@ Initiator                    Responder
    |<----------------------------|
    |   ack                       |
    |---------------------------->|
-   |   [ file transfer ]         |
+   |   [optional further         |
+   |    negotiation]             |
+   |<--------------------------->|
+   |   File Transfer             |
    |============================>|
-   |   session-terminate         |
-   |<----------------------------|
-   |   ack                       |
-   |---------------------------->|
-   |                             |
-  ]]></code>
-  <p>First the initiator sends a Jingle session-initiate.</p>
-  <example caption="Initiator sends session-initiate"><![CDATA[
-<iq from='romeo@montague.lit/orchard'
+   |                             |]]></code>
+
+  <section2 topic='Offering a File' anchor='offering'>
+    <p>To start a File Offer, the initiator sends a Jingle session-initiation request to a potential responder. The request specifies three things:</p>
+    <ol>
+      <li>A content 'senders' attribute with the value of 'initiator' to indicate this is a File Offer.</li>
+      <li>An application type of "urn:xmpp:jingle:apps:file-transfer:4". In particular, the &lt;description/&gt; element contains a &lt;file/&gt; elements describing the file to be sent.</li>
+      <li>An appropriate transport method.</li>
+    </ol>
+    <p>In this example, the initiator is &lt;romeo@montague.example&gt;, the responder is &lt;juliet@capulet.example&gt;, the application type is a File Offer, and the transport method is jingle-s5b (XEP-0260).</p>
+    <p>The flow is as follows.</p>
+    <p>First the initiator sends a Jingle session-initiate.</p>
+    <example caption="Initiator sends session-initiate"><![CDATA[
+<iq from='romeo@montague.example/dr4hcr0st3lup4c'
     id='nzu25s8'
-    to='juliet@capulet.lit/balcony'
+    to='juliet@capulet.example/yn0cl4bnw0yr3vym'
```

Comment:
Nit: I would prefer the human readable balcony/orchard as resource string over this.